collect_app/src/main/java/org/odk/collect/android/logic/FormController.java

```diff
@@ -1151,6 +1158,7 @@ public InstanceMetadata getSubmissionMetadata() {
 
         String instanceId = null;
         String instanceName = null;
+        boolean audit = false;
```

Comment:
Is this true of local variables?  I get confused by when you have to initialise variables in Java.  I note that the String have been initialised to null previously.  Is that required?